docs/pages/blog/benny-joo-joining.md

```diff
@@ -17,6 +17,6 @@ He had a great start with his internship, implementing the [Masonry component](h
 As part of the Core team, he will help with the community support, as well as work on some of the new features we plan for the community version of the library, like the unstyled components and the second design system.
 I am excited to see what other valuable contributions he will make in the future.
 
-The Core team will continue developing the foundations on which to build great design systems. Here's a quick view of [the community roadmap](https://github.com/orgs/mui/projects/18/views/1).
+The Core team will continue developing the foundations on which to build great design systems.
```

Comment:
For these two posts (this one and the above), ended up finding it simpler to just delete the roadmap mention as there's no "community roadmap" anymore (which was a Material UI community roadmap instead), rather than link to the Projects tab again. Seems like not a big deal, as I doubt there's any access to these pages anymore 😅